lib/ansible/modules/cloud/ovirt/ovirt_vms.py

```diff
@@ -232,6 +232,20 @@
             - "C(nic_name) - Set name to network interface of Virtual Machine."
             - "C(nic_on_boot) - If I(True) network interface will be set to start on boot."
         version_added: "2.3"
+    kernel_path:
+        description:
+            - "Path to kernel image to boot the virtual machine."
+            - "Kernel image must be stored on either the ISO domain or on the host's storage."
+        version_added: "2.3"
+    initrd_path:
+        description:
+            - "Path to a ramdisk to be used with the kernel."
+            - "Ramdisk image must be stored on either the ISO domain or on the host's storage."
+        version_added: "2.3"
+    kernel_params:
+        description:
+            - "Kernel command line parameter strings to be used with the defined kernel on boot."
```

Comment:
I'd rephrase: "Kernel command line parameters (formatted as string) to be used with the kernel specified by `kernel_path` option."